Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj

```diff
@@ -99,6 +99,15 @@
 		5EE408551E8417E3002A6697 /* CreateMealOccasionsViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5EE408531E8417E3002A6697 /* CreateMealOccasionsViewController.xib */; };
 		5EF436481EA7B5A300D9ABD0 /* MockLocationManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5EF436471EA7B5A300D9ABD0 /* MockLocationManager.swift */; };
 		93D4F766276CC765FC6209A7 /* Pods_MealTrackingPilotTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F9563EBA0AC2B40CED071536 /* Pods_MealTrackingPilotTests.framework */; };
+		9807AD1323291FC700D56EB9 /* UITextView+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0A23291FC700D56EB9 /* UITextView+Extensions.swift */; };
+		9807AD1423291FC700D56EB9 /* Foundation+Helper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0B23291FC700D56EB9 /* Foundation+Helper.swift */; };
+		9807AD1523291FC700D56EB9 /* CGRect+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0C23291FC700D56EB9 /* CGRect+Extensions.swift */; };
+		9807AD1623291FC700D56EB9 /* UIViewController+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0D23291FC700D56EB9 /* UIViewController+Extensions.swift */; };
+		9807AD1723291FC700D56EB9 /* TypeIdentifiable+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0E23291FC700D56EB9 /* TypeIdentifiable+Extension.swift */; };
+		9807AD1823291FC700D56EB9 /* CGPoint+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0F23291FC700D56EB9 /* CGPoint+Extension.swift */; };
```

Comment:
Change the file name to CGPoint+Extensions.swift

Reply:
Done changes